src/main/clm_instMod.F90

```diff
@@ -383,6 +385,13 @@ subroutine clm_instInit(bounds)
        if (decomp_method == century_decomp ) then
           call init_decompcascade_bgc(bounds, soilbiogeochem_state_inst, &
                                       soilstate_inst )
+       else if (decomp_method == mimics_decomp ) then
+          call init_decompcascade_mimics(bounds, soilbiogeochem_state_inst, &
+                                         soilstate_inst)
+       else
+          write(iulog,*) 'WARNING:'
```

Comment:
I'd actually remove the else and warning here, as it's already done in decomp_cascase_par_init.

Reply:
Removed `else`